component/src/main/java/org/wso2/extension/siddhi/store/rdbms/util/RDBMSTableConstants.java

```diff
@@ -70,6 +70,30 @@
     public static final String ANNOTATION_ELEMENT_FIELD_LENGTHS = "field.length";
     public static final String ANNOTATION_ELEMENT_POOL_PROPERTIES = "pool.properties";
     public static final String ANNOTATION_ELEMENT_JNDI_RESOURCE = "jndi.resource";
+    public static final String ANNOTATION_DRIVER_CLASS_NAME = "jdbc.driver.name";
+
+    //Configurable System Parameters
+    public static final String PROPERTY_SEPARATOR = ".";
+    public static final String MIN_VERSION = "minVersion";
+    public static final String MAX_VERSION = "maxVersion";
+    public static final String TABLE_CHECK_QUERY = "tableCheckQuery";
+    public static final String TABLE_CREATE_QUERY = "tableCreateQuery ";
```

Comment:
This is typo error, fixed by the commit : 45231931dc3b2437fac8359ed347a1caa8eec569